src/chrome/content/scaffold/scaffold.js

```diff
@@ -362,13 +360,18 @@ var Scaffold = new function() {
 				var typeNames = Zotero.ItemTypes.getTypes().map(t => t.name);
 				document.getElementById('output').value = JSON.stringify(typeNames, null, '\t');
 				break;
+			case "shortcuts":
+				var value = Zotero.File.getContentsFromURL(`chrome://scaffold/content/templates/shortcuts.txt`);
```

Comment:
Did you forget to commit this file?

Reply:
Yes, sorry for that. Now, I add this missing file and solved the merge conflicts before. Please have a look at the new version.